index.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
```

Comment:
Instead of implementing everything in a single file, you can make abstractions and follow the single responsibility principle where each function is responsible for one specific thing. For example, one file for defining all API routes, one handler file for all custom logic, and one data access layer file which is responsible for interacting with your database (JSON file in this case)